### DISCUSSION TITLE
Only push selected Z-Wave parameters to MQTT server

### DISCUSSION THREAD

**Discussion by Thomas-Magnum** [zwave-js/zwave-js-ui#2969] [Q&A] [answered]
Hi,
I'm pretty new to MQTT, so sorry for potentially incorrect terminology.

I've a pretty large Z-Wave network and have been able to setup zwave-js-ui and communication to my MQTT-server. However due to the large number of Z-Wave devices the MQTT-server gets swamped with status updates for thousands of different topics.
Is there any way to limit this, i.e. can I enable only specific parameters/variables from my Z-Wave devices to be pushed to the MQTT server?

Thanks!

**Comment by robertsLando** [comment, marked as answer]
Sure! You can specify the values and also a custom topic for them from [devices values table](https://zwave-js.github.io/zwave-js-ui/#/usage/setup?id=device-values-configuration) then remember to set [gateway type](https://zwave-js.github.io/zwave-js-ui/#/usage/setup?id=gateway) to configured manually

**Reply by Thomas-Magnum** [reply]
Excellent, thanks!
